README.md

```diff
@@ -3,12 +3,24 @@ Yeast
 
 ## Summary
 
+*Yeast contains any MAS CSS that is shared between projects.*
+
 1. Shared SCSS mixins, variables and functions for MAS apps and engines - these do not output any CSS when included in a project.
 
 2. SCSS partials that were originally within the MAS Website (frontend repo). Since we have a need to create standalone MAS branded tools that do not inherit the main MAS Website CSS we have moved the reusable SCSS partials into Yeast to avoid duplicating across projects.
 
 Yeast should be loaded into new projects as a Bower component and then the SCSS partials pulled in to the main project stylesheet.
 
+## Bower
+
+As of 7/11/2016 Yeast is registered with the Bower registry so we can use semantic versioning to reference it as a Bower package, rather than pulling straight from GitHub as we were earlier.
+
+When updating Yeast, update the version in bower.json, and also _create a Git tag_ with the same version:
+
+`git tag -a v1.0.0 -m "Initial tag for Bower"`
+
+This will make the version available in the Bower registry.
+
```

Comment:
Should we also add the necessary changes that need to be made in frontend and other repos that depend on Yeast here?